docs/_macros/compare.md

```diff
@@ -1,20 +1,14 @@
 ---
 layout: default
 class: Macro
-title: compare NUMBER NUMBER
-summary: Compare two numbers. 0 is equal, 1 means a > b, -1 is a < b.
+title: compare STRING STRING
+summary: Compare two strings by using the String method compareTo. <astring>.compareTo(<bstring>)
 ---
 
-	static String	_ncompare	= "${ncompare;<anumber>;<bnumber>}";
+	static String	_compareHelp  = ${compare;<astring>;<bstring>}";
```

Comment:
This should be replaced with real text :-)

The original population of the macro pages was simple coping-in of the implementation code. The code here is probably not even the real code any more.